### DISCUSSION TITLE
Feature Request:  Placing objects on ceiling

### DISCUSSION THREAD

**Discussion by buddy90210** [google/model-viewer#3450] [Q&A]
Is it possible to automatically determine the ceiling using hittest? Will only the direction of the normal vector to the surface change?

**Comment by elalish** [comment]
Are you asking a WebXR question or a model-viewer question?

**Reply by buddy90210** [reply]
WebXR

**Reply by elalish** [reply]
Then yes, you're correct. I would hope there's a better forum somewhere to ask about the WebXR API, but I don't know one off the top of my head...
